docs/how-tos/entanglement_forging/use-asymmetric-bitstrings.ipynb

```diff
@@ -71,23 +63,23 @@
    "outputs": [],
    "source": [
     "# Define a molecular system of interest - Methyl radical\n",
-    "molecule = Molecule(\n",
-    "    geometry=[\n",
-    "        [\"C\", [0.0, 0.0, 0.00]],\n",
-    "        [\"H\", [1.0790, 0.0, 0.00]],\n",
-    "        [\"H\", [-0.5395, -0.9344, 0.00]],\n",
-    "        [\"H\", [-0.5395, 0.9344, 0.00]],\n",
+    "molecule = MoleculeInfo(\n",
```

Comment:
Oh interesting: following up on my last comment, you _did_ use `MoleculeInfo` here.